pceplib/pcep_msg_objects.h

```diff
@@ -385,11 +385,16 @@ struct pcep_object_lsp {
 	bool flag_c;
 };
 
+#define ENTERPRISE_NUMBER_CISCO 9
+#define ENTERPRISE_COLOR_CISCO 65540
 /* RFC 7470 */
 struct pcep_object_vendor_info {
 	struct pcep_object_header header;
 	uint32_t enterprise_number;
 	uint32_t enterprise_specific_info;
+	uint32_t enterprise_specific_info1; /* cisco sends color for PcInit */
+	uint32_t enterprise_specific_info2;
+	uint32_t enterprise_specific_info3;
```

Comment:
I think that a pointer to void i.e. `void *tlvs` will be better. And following the entreprise number you could cast the pointer to the expected structure / type

Reply:
I would prefer this way because in fact the pceplib is able to extract tlv's based in the tlv id but in this case there is not tlv id. So I consider it part of the object not part of the tlv.